02-spatial-data.Rmd

```diff
@@ -253,36 +269,38 @@ plot(world_centroids, add = TRUE, cex = world$pop / 1e8, lwd = 3)
+For this reason we cover each currently supported `sfg` type in the next subsections before moving on to describe how these can be combined to form `sfc` and eventually full `sf` objects.
 
 #### Simple feature geometry types {#geometry}
 
 <!-- This section demonstrates how the full range of geometry types supported by the **sf** package can be created, combined and plotted. -->
 Geometries are the basic building blocks of simple features.
-Simple features could be represented as one of the 17 geometry types using the **sf** package.
-In this chapter we will focus on seven, the most commonly used, simple features types: `POINT`, `LINESTRING`, `POLYGON`, `MULTIPOINT`, `MULTILINESTRING`, `MULTIPOLYGON` and `GEOMETRYCOLLECTION`.<!--FIG-->
-The whole list of possible feature types could be found in [the PostGIS manual ](http://postgis.net/docs/using_postgis_dbmanagement.html).
-
-Simple features, outside of an R environment, could be represented in one of two ways, ether as a well-known binary (WKB) or well-known text (WKT). 
-Well-known binary (WKB) representations are usually hexadecimal strings, which are used to transfer and store geometry objects in databases.
-Well-known text (WKT), on the other hand, is a text markup description of simple features. 
-Both formats are exchangeable, therefore we would focus only on the well-known text (WKT) representation.
-
-All of the geometry types are built of points. 
-Each point could be described as coordinates in a 2D, 3D or 4D space:
-
-- X and Y coordinates represent location of points. It could be either easting and northing or longitude and latitude.
-- A Z coordinate denotes altitude.
+Simple features in R can take on one of the 17 geometry types supported by the **sf** package.
+In this chapter we will focus on the seven most commonly used types: `POINT`, `LINESTRING`, `POLYGON`, `MULTIPOINT`, `MULTILINESTRING`, `MULTIPOLYGON` and `GEOMETRYCOLLECTION`.<!--FIG-->
+Find the whole list of possible feature types in [the PostGIS manual ](http://postgis.net/docs/using_postgis_dbmanagement.html).
+
+Generally, well-known binary (WKB) or well-known text (WKT) are the standard encoding for simple feature geometries.
+WKB representations are usually hexadecimal strings easible readable for computers.
+This is why, GIS and spatial databases use WKB to transfer and store geometry objects.
+WKT, on the other hand, is a human-readable text markup description of simple features. 
+Both formats are exchangeable, and if we present one, we will naturally choose the WKT representation.
+
+The basis for each of the geometry types is the point. 
+A point is simply a coordinate in 2D, 3D or 4D space:
+
+- X and Y coordinates represent the location of the points on the surface of the Earth. Depending on the coordinate reference system you use, this can be, for example, easting and northing or longitude and latitude.
```

Comment:
Good point. In fact **sf** is *not good* at dealing with data in geographic (on the surface of the Earth) CRSs because GEOS itself assumes a planar projection. The overall direction is good though. I suggest:

```
- X and Y coordinates represent the location points in 2D space, with units depending on the coordinate reference system (CRS) of the geometry data.
Commonly used units include easting and northing in meters or longitude and latitude (see Chapter 6). 
```

We can update the cross-reference when chapter 6 goes 'live'!